_posts/2022-08-05-hacking-for-cruise-control-ui.md

```diff
@@ -0,0 +1,279 @@
+---
+layout: post
+title: "Hacking Strimzi for Cruise Control UI"
+date: 2022-10-31
+author: kyle_liberti
+---
+
+For some time now, Strimzi has used for [Cruise Control](https://github.com/linkedin/cruise-control) for automated partition rebalancing and has offered Strimzi custom resources for safely configuring, deploying, and communicating with Cruise Control through the Strimzi Operator.
+These custom resources not only provide a strong abstraction leaving Cruise Control as an implementation detail but also offer great declaritive support.
```

Comment:
The "strong abstraction" isn't true: The `Rebalance` CR doesn't really hide the fact that it's CC underneath at all.

```suggestion
These custom resources provide great support for declarative rebalancing.
```

Even this is pretty weird, since rebalancing itself is inherently imperative. It's only with support for automatic rebalancing that the declarative aspect becomes meaningful.